src/app/features/location/location-input/location-input.component.ts

```diff
@@ -75,6 +75,7 @@ export class LocationInputComponent extends CustomFormControlDirective<GeoLocati
 
     const ref = this.dialog.open(MapPopupComponent, {
       width: "90%",
+      height: "100vh",
```

Comment:
```suggestion
      height: "95vh",
```
let's give this a small margin at least, for better design